src/pages/code-for-good.js

```diff
@@ -0,0 +1,327 @@
+    size,
+} from '../components/dev-hub/theme';
+import { H3, ArticleH2, ArticleH4, P, P3 } from '../components/dev-hub/text';
+import styled from '@emotion/styled';
+import Button from '../components/dev-hub/button';
+import HeroBanner from '../components/dev-hub/hero-banner';
+import BackgroundWaves from '../images/code-for-good/background-waves.png';
+import WildAidLogo from '../images/code-for-good/wild-aid-logo.png';
+import WildAidDashboard from '../images/code-for-good/wild-aid-dashboard.png';
+import VesselRecord from '../images/code-for-good/vessel-record.png';
+import Link from '../components/dev-hub/link';
+
+const CAPTION_MAX_WIDTH = '345px';
+const BODY_TEXT_MAX_WIDTH = '320px';
+const HERO_TEXT_WIDTH = '425px';
+const DASHBOARD_IMAGE_WIDTH = 780;
+const VESSEL_IMAGE_WIDTH = 345;
+const LOGO_IMAGE_WIDTH = 350;
+const LOGO_IMAGE_HEIGHT = 450;
+
+const Header = styled('header')`
+    display: flex;
+    justify-content: space-between;
+    padding-top: ${size.large};
+    @media ${screenSize.upToLarge} {
+        flex-direction: column-reverse;
+        margin-bottom: ${size.large};
+    }
+`;
+const HeaderText = styled('div')`
+    margin-bottom: -${size.xlarge};
+    max-width: ${HERO_TEXT_WIDTH};
+    @media ${screenSize.upToMedium} {
+        width: 100%;
+    }
+`;
+
+const HeaderLink = styled(Link)`
+    color: ${colorMap.darkGreen};
+    display: inline;
+    margin-top: ${size.large};
+    margin-left: ${size.mediumLarge};
+    white-space: nowrap;
+    @media ${screenSize.upToLarge} {
+        display: block;
+        margin: 0;
+    }
+`;
+
+const StyledHeroBanner = styled(HeroBanner)`
+    margin: 0 ${size.medium};
+    @media ${screenSize.upToLarge} {
+        margin: 0;
+    }
+`;
+
+const StyledP = styled(P)`
+    margin-top: ${size.articleContent};
+    margin-bottom: ${size.xlarge};
+`;
+
+const BodyText = styled(P)`
+    margin-top: ${fontSize.default};
+`;
+
+const StyledButton = styled(Button)`
+    margin-top: ${size.large};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.medium};
+    }
+`;
+
+const BodyContent = styled('div')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    width: 100%;
+    @media ${screenSize.upToMedium} {
+        padding: 0 ${size.default};
+        width: 100%;
+    }
+`;
+
+const StyledParagraphLink = styled(Link)`
+    color: ${colorMap.darkGreen};
+    text-decoration: none;
+`;
+
+const StyledLink = styled(Link)`
+    color: ${colorMap.darkGreen};
+    font-weight: bold;
+    margin-top: ${size.mediumLarge};
+    text-decoration: none;
+`;
+
+const UnderstandTextSection = styled('div')`
+    margin-bottom: ${size.xlarge};
+    margin-top: ${size.xlarge};
+    max-width: ${BODY_TEXT_MAX_WIDTH};
+    @media ${screenSize.upToMedium} {
+        width: 100%;
+    }
+`;
+
+const VesselAppTextContainer = styled('div')`
+    margin-bottom: ${size.xlarge};
+    max-width: ${BODY_TEXT_MAX_WIDTH};
+    @media ${screenSize.upToMedium} {
+        width: 100%;
```

Comment:
I think for firefox and safari we also need to set max-width to `100%`  as well here